src/DataSets/DataSets.component.js

```diff
@@ -150,6 +173,16 @@ const DataSets = React.createClass({
         this.setState({settingsOpen: false});
     },
 
+    openLogs() {
+        this.context.d2.dataStore.get('dataset-configuration').then(store => {
```

Comment:
We can move this logic to the `log.js` module so the implementation is hidden and can be easily reused.